pkg/cmd/extension/command.go

```diff
@@ -219,6 +221,33 @@ func NewCmdExtension(f *cmdutil.Factory) *cobra.Command {
 				return nil
 			},
 		},
+		&cobra.Command{
+			Use:   "run <name> [args]",
+			Short: "Run an installed extension",
+			Long: heredoc.Doc(`
+			Run an extension using either the extension name, or the extension repository's
+			organization and name. For example, if the extension repository is
+			"owner/gh-extension", you can pass either "extension" or "owner/gh-extension".
+
+			Use the extension repository's organization and name to disambiguate extensions
+			that have the same name, or share a name with a built-in command.
+
+			All arguments after the extension name will be forwarded to the executable
+			of the extension.
+			`),
+			Example: heredoc.Doc(`
+			$ gh extension run extension
+			$ gh extension run owner/gh-extension
+			`),
+			Args: cobra.MinimumNArgs(1),
+			RunE: func(cmd *cobra.Command, args []string) error {
+				if found, err := m.Dispatch(args, os.Stdin, os.Stdout, os.Stderr); !found {
+					return fmt.Errorf("command %s not found", args[0])
```

Comment:
Nit: let's say "extension foo not found" so that it's clear that you might have misspelled the name of an extension. Also we could put the extension name in quotes so that it stands out in the rest of the sentence